app/screens/home/search/results/empty_messages.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import Svg, {Path, Ellipse} from 'react-native-svg';
+
+function EmptyMessages() {
```

Comment:
does this component differ from the one we already have under no result with term?